run.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+git pull
+. .env
+cd ui
+yarn
+node_modules/.bin/ng build
```

Comment:
with `--prod` the static files get random name, which breaks the flask static file serving currently.

We have to fix that first, IMHO not part of this PR.